scripts/ci.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
```

Comment:
I'd recommending a `set -e` to ensure fast failure. Without it we could upload a build to riffraff with a broken js build, for example.

Reply:
Good shout! That's been added now.